### ISSUE TITLE
Debian package depencies

### ISSUE THREAD

**Issue by oOorel** [Brewtarget/brewtarget#890]
libqt6gui6t64 libqt6network6t64 libqt6printsupport6t64 libqt6sql6t64 libqt6widgets6t64 packages are not available in Debian.

Maybe there are problems i didn't noticed, but Brewtarget 4.X seems to work ok in Debian with libqt6gui6 libqt6network6 libqt6printsupport6 libqt6sql6 libqt6widgets6 packages installed, even if it breaks the package system because of the depencies.

Maybe the solution to have the .deb package compatible with both Ubuntu and Debian dists could be to have someting like libqt6gui6 | libqt6gui6t64 in the package depencies?

libqt6gui6t64 package provides libqt6gui6... maybe a better solution is to change the depencies of the Brewtarget package so its requires libqt6 non-t64?

**Comment by matty0ung** [comment]
We started out just depending on the non-t64 versions of the libraries and then I changed it to fix https://github.com/Brewtarget/brewtarget/issues/840.  I think eventually all these libraries will be t64 everywhere because the changes are to do with fixing the 'year 2038 problem' (per https://wiki.debian.org/ReleaseGoals/64bit-time).  But, you're right, that they are not all yet available in stable versions of Debian.  (Looks like a lot of them are only in Debian Unstable/Sid for now, not Debian Bookworm/Stable.)

I'll have a go at doing `libqt6gui6 | libqt6gui6t64` etc.  It would be nice for the dependencies to work on both Debian and Ubuntu.
